Microsoft.Toolkit.Uwp.SampleApp/SamplePages/MarkdownTextBlock/MarkdownTextBlockPage.xaml

```diff
@@ -0,0 +1,47 @@
+﻿<Page
```

Comment:
@ScottIsAFool Didn't know this thing exists. @QuinnDamerell that is a VS Extension to install, named "XAML Styler"